app/views/coronavirus_form/living_with_vulnerable.html.erb

```diff
@@ -1 +1,4 @@
-<%= render partial: "question_radio_buttons", locals: {question: "living_with_vulnerable"} %>
+<%= render partial: "question_radio_buttons", locals: {
+  question: "living_with_vulnerable",
+  group: "going_in_to_work",
```

Comment:
this should be ` living_with_vulnerable`

Reply:
The question or the group?

Reply:
the group

Reply:
ooh sorry I am wrong, I misread